tests/test_skip_behavior.py

```
@@ -0,0 +1,54 @@
+    timeout=60,
+)
+if "PASSED" in result.stdout:
+    print("   ✓ Test RAN and PASSED (correct)")
+elif "SKIPPED" in result.stdout:
+    print("   ✗ Test was SKIPPED (incorrect - should run with -n0)")
+else:
+    print(f"   ? Unexpected result: {result.stdout[:200]}")
+
+# Test 2: Run with -n2 (should skip)
+print("\n2. Running with -n2 (parallel mode - should SKIP)...")
+result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)